GenericDiffractometer.py

```diff
@@ -185,6 +208,10 @@ def __init__(self, name):
              GenericDiffractometer.CENTRING_METHOD_MOVE_TO_BEAM: \
                  self.start_move_to_beam}
 
+        self.connect(self, 'equipmentReady', self.equipment_ready)
```

Comment:
Hi Ivars,

Thanks! I was planning to do a second PR with more signal related. Somehow, this one slipped through my eyes. I will fix it.


From: Ivars Karpics [mailto:notifications@github.com]
Sent: Thursday, April 14, 2016 11:45 AM
To: mxcube/HardwareObjects
Cc: Jie Nan
Subject: Re: [mxcube/HardwareObjects] Improvement of Genericdiffractometer (#31)


In GenericDiffractometer.py<https://github.com/mxcube/HardwareObjects/pull/31#discussion_r59689325>:

> @@ -185,6 +208,10 @@ def __init__(self, name):

>               GenericDiffractometer.CENTRING_METHOD_MOVE_TO_BEAM: \

>                   self.start_move_to_beam}

>

> +        self.connect(self, 'equipmentReady', self.equipment_ready)

Hi @JieNanMAXIV<https://github.com/JieNanMAXIV> and other

I accepted this PR without testing, but already now see a issue.
equipment_ready is not defined in GenericDiffractometer (it is in MAXIV/BIOMAXMiniDiff.py)
Should we add this method to HardwareObject in HardwareRepository?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/mxcube/HardwareObjects/pull/31/files/df567d5a90b84782670a548677e8561685fe5adc#r59689325>